homework/fibonacci/fibonacci.hpp

```diff
@@ -2,10 +2,37 @@
 
 int fibonacci_iterative(int sequence) {
     // TODO: Your implementation goes here
+        int an1 = 1; // a(n-1) - n-1 element of fibonacci series
+    int an2 = 0; // a(n-2) - n-2 element of fibonacci series
+    int an = 0; // a(n) - n element of fibonacci series
```

Comment:
What is `an` ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...